.github/actions/cl2-modules/egw/config.yaml

```diff
@@ -1,5 +1,16 @@
+{{$TestImage := .CL2_EGW_TEST_IMAGE}}
```

Comment:
nit: you could move it to each separate yaml so you don't need to pass it around everywhere. 
So for example in `manifests/external-target-pod.yaml` just have:
```
{{$TestImage := .CL2_EGW_TEST_IMAGE}}
```
but that's just up to your preference.